CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 ## Unreleased
 
 * [BUGFIX] Fix inaccuracy in `e2ealerting` caused by invalid purging condition on timestamps. #117
-
+* [CHANGE] When using `rules` commands, cortex ruler API requests will now default to using the `/api/v1/` prefix. 
+The `--use-legacy-routes` flag has been added to allow users to use the original `/api/prom/` routes. #99
```

Comment:
This should be a single line
```suggestion
* [CHANGE] When using `rules` commands, cortex ruler API requests will now default to using the `/api/v1/` prefix. The `--use-legacy-routes` flag has been added to allow users to use the original `/api/prom/` routes. #99
```

Reply:
I have updated, now this is a single line.